pkg/controllers/edgeconnect/conditions.go

```diff
@@ -0,0 +1,4 @@
+package edgeconnect
+
+// SecretConfigConditionType identifies the secret config condition.
+const SecretConfigConditionType = "SecretConfigConditionType"
```

Comment:
why is this not in `pkg/controllers/edgeconnect/consts/consts.go`?

IMO, the structure of the `edgeconnect` package is a bit messy(or just doesn't follow what we do elsewhere),because there is only 1 BIG reconciler, that handles everything

Reply:
Yeah, I'm aware of it, and we will refactor it soon to sync with the dynakube structure.